odev/commands/database/create.py

```diff
@@ -91,6 +91,11 @@ def ensure_database_not_exists(self):
         """Drop the database if it exists."""
         with self._database.psql().nocache():
             if self._database.exists:
+
+                process = self.odoobin or OdoobinProcess(self._database)
+                if process.is_running:
+                    raise self.error(f"Database {self._database.name!r} is already running")
```

Comment:
We should be able to abstract the process part as it's already handled by the database class.
I would also put a more descriptive message telling the user to shut it down before retrying
```suggestion
                if self._database.running:
                    raise self.error(f"Database {self._database.name!r} is running, stop it and try again")
```